components/alert.html

```diff
@@ -0,0 +1,150 @@
+    <title>Docs</title>
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/fontawesome.min.css" integrity="sha512-OdEXQYCOldjqUEsuMKsZRj93Ht23QRlhIb8E/X0sbwZhme8eUw6g8q7AdxGJKakcBbv7+/PX0Gc2btf7Ru8cZA==" crossorigin="anonymous" /> -->
+    
+    <link href="/main.css" rel="stylesheet" />
+    <link href="/css/alert.css" rel="stylesheet" />
+    
+</head>
+<body>
+    <div id="page">
+      <header>
+        <div class = "header">
+            
+        <h1  class="head-logo-padd head-color" id="logo">
+        AASAN-UI</h1>
+        <h3  class="head-padd head-color">
+        <a class="link" href="/index.html">Home</a></h3>
+        <h3 class="head-padd ">Docs</h3>
+        <h3  class="head-padd head-color">
+        <a class="link" href="/about.html">About</a></h3>
+        </header>
+
+          
+        <nav>
+          <ul>
+            <div class="nav-li head-color-active">
+                    Alert 
+            </div>
+            <div class="nav-li">
+              <a class="link" href="/components/avatar.html">
+                Avatar</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/badge.html">
+                Badge</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/button.html">
+                Button</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/card.html">
+                Card</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/image.html">
+                Image</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/input.html">
+                Input</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/list.html">
+                List</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/modal.html">
+                Modal</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/navigation.html">
+                Navigation</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/rating.html">
+                Rating</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/toast.html">
+                Toast</a></div>
+            <div class="nav-li">
+              <a class="link" href="/components/typography.html">
+                Typography</a></div>
+            
+          </ul>
+          </nav>
+
+
+        <main>
+       <div>
+         <div class="vl">
+         
+         <span>
+          <h1 class="main-heading-center">Alert</h1>
+          
+
+          <div class="success b">
+            <p><i class="far fa-check-circle"> 
+              This is success alert</i></p>
+          </div>
+          
+          <div class="failure b">
```

Comment:
what is b?

Reply:
Changed it , actually I had used it for margin